load-to-globalsearch.mlcp

```diff
@@ -0,0 +1,36 @@
+#-----------------------------------------------------------------------------#
+# An mlcp option file to push the _pages folder into the global search dhf
+# instance. Use as follows (assumes mlcp as a globally configured executable):
+#
+# mlcp -options_file load-to-globalsearch.mlcp -username user -password pass
+#
+# Use your network credentials for `user` and `pass`. VPN connection is
+# required if outside the company network.
+#-----------------------------------------------------------------------------#
+import
+-input_file_path
+_pages/*
+-document_type
+text
+-output_uri_replace
+".*/marklogic-data-hub,'/marklogic-data-hub'"
+-fastload
+false
+-host
+globalsearch.marklogic.com
+-port
+8011
+-modules_root
+/
+-transform_module
+"/MarkLogic/data-hub-framework/transforms/mlcp-flow-transform.xqy"
```

Comment:
This is the 3.0 URL, and will probably start failing at some point.  I'd recommend being on DHF 4+ and using /data-hub/4/transforms/mlcp-flow-transform.xqy

Reply:
You might also want to use port 8010.  I'd avoid using the FINAL appserver for anything related to DHF as the is the part that does not align with DHS